Python Guide/.ipynb_checkpoints/Python Kaggle Guide (Titantic)-checkpoint.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
This is in the `.ipynb_checkpoints` which you should include in the `.gitignore`